### PR TITLE
Emoji 5.0 Unicode 10.0 🤩

### PR DESCRIPTION
**Pull request by Francescu**

### CODE REVIEW

**Comment by Francescu** [comment]
Should fix #5

**Comment by onmyway133** [comment]
@Francescu This is great. Thanks 👍

File: Sources/Emoji.swift [hunk 1]
```diff
@@ -1431,4 +1431,64 @@ public let emojiList: [String: String] = [
   "yemen": "🇾🇪",
   "zambia": "🇿🇲",
   "zimbabwe": "🇿🇼",
-]
\ No newline at end of file
+  "starstruck": "🤩",
```
Comment (onmyway133) [inline Sources/Emoji.swift]: @Francescu This is cool. However I would like to move these manual added emojies into a new file called ManualAddedEmojis. As this `Emoji.swift` file is generated by a script

**Comment by Francescu** [comment]
Actually I generated it with your parse script using an different JSON URL, that I found on a fork of the repo you were using.

Francescu

**Pull request merged**